nuxeo-compound-documents-web/nuxeo-compound-documents.html

```diff
@@ -14,3 +14,44 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
+<!-- Disable blob buttons -->
+<nuxeo-slot-content name="replaceBlobAction" slot="BLOB_ACTIONS" order="35" disabled>
+  <template>
+    <nuxeo-replace-blob-button document="[[document]]" xpath="[[xpath]]"></nuxeo-replace-blob-button>
+  </template>
```

Comment:
This is not needed
```suggestion
```

Reply:
Ah, you read my mind. I was going to ask if I should keep the actual content of the contribution just for reference, or I should remove it. Thanks